vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
 # Content Types
 
```

Comment:
Might want to add the sidebarDepth: 2 to the header to make the page a bit more usable.